samples/snippets/train_processor_version_sample.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START documentai_train_processor_version]
+
+from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import FailedPrecondition, NotFound
+from google.cloud import documentai_v1beta3 as documentai
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID'
+# processor_version_display_name = 'new-processor-version'
+# train_data_uri = 'gs://bucket/directory/' # (Optional)
+# test_data_uri = 'gs://bucket/directory/' # (Optional)
+
+
+def train_processor_version_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version_display_name: str,
+    train_data_uri: str = None,
+    test_data_uri: str = None,
+):
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+    opts = ClientOptions(api_endpoint=f"{location}-documentai.googleapis.com")
+
+    client = documentai.DocumentProcessorServiceClient(client_options=opts)
+
+    # The full resource name of the processor
+    # e.g.: projects/project_id/locations/location/processors/processor_id
+    parent = client.processor_path(project_id, location, processor_id)
+
+    processor_version = documentai.ProcessorVersion(
+        display_name=processor_version_display_name
+    )
+
+    # If Train/Test data is not supplied, the default sets in the Cloud Console will be used
```

Comment:
Replace with "train/test" -- I don't think these are proper nouns in this context.